runner/src/server/plugins/engine/views/components/freetextfield.html

```diff
@@ -0,0 +1,96 @@
+{% macro FreeTextField(component) %}
+
+<head>
+  <script src="/assets/tinymce.min.js"></script>
+  <script type="text/javascript">
+
+    tinymce.init({
+      setup: function (editor) {
+        editor.on('init', function () {          
+          setWordCount(editor);
+        });
+        editor.on('input', function (e) {          
+          setWordCount(editor);
+        });
+        editor.on('setcontent', function(e) {
+          // When pasting into the field, update the word count         
+          setWordCount(editor);
+        });
```

Comment:
This in theory might work, from what I've seen in other libraries, but not sure:

```suggestion
      setup: function (editor) {
        editor.on('init input setcontent', function () {          
          setWordCount(editor);
        });
```